pandas/tests/generic/test_generic.py

```diff
@@ -159,27 +159,14 @@ def test_downcast(self):
 
         o = self._construct(shape=4, value=9, dtype=np.int64)
         result = o.copy()
-        result._data = o._data.downcast(dtypes="infer")
+        result._data = o._data.downcast()
         self._compare(result, o)
 
-        o = self._construct(shape=4, value=9.0)
```

Comment:
these are legit tests, no?

Reply:
they pass a dtypes kwarg that this PR removes